### PR TITLE
Closure compatibility

### PR DESCRIPTION
**Pull request by benmcginnis**
Working version, though definitely needs to be refactored.

### CODE REVIEW

**Comment by benmcginnis** [comment]
Also, I couldn't get the regex based `{lb}{rb}` escaping to work because of the problem of media queries creating an outer capture group that wasn't getting picked up.  If we can get that to work it would definitely be preferred.

File: tasks/inline.js [hunk 1]
```diff
@@ -47,17 +47,51 @@ module.exports = function(grunt) {
...
+		}
+		return datauri(absoluteUrl.replace(/\?.*$/, ''));
+	}
+
 	function isRemotePath( url ){
 		return url.match(/^'?https?:\/\//) || url.match(/^\/\//);
 	}
 
+	function soyEscape(contents) {
+		return contents.replace(/(\{([^}]+)\})/gmi, '{lb}$2{rb}');
```
Comment (benmcginnis) [inline tasks/inline.js:92]: @mhupman this was the part that was sooo close to working.

Comment (mhupman) [inline tasks/inline.js:92]: `.selector {content: '{';}` 💣

Comment (mhupman) [inline tasks/inline.js:92]: Also, why not just replace `{` with `{lb}` and `}` with `{rb}`?

**Comment by mhupman** [comment]
lgtm

File: tasks/inline.js (hunk 1 above)
Comment (benmcginnis) [inline tasks/inline.js:92]: i figured that would create a cycle where i'd get `{lb{rb}` right?

**Pull request merged**